packages/cli/src/dbt/targets/athena.ts

```diff
@@ -0,0 +1,82 @@
+import {
+    CreateAthenaCredentials,
+    ParseError,
+    WarehouseTypes,
+} from '@lightdash/common';
+import { JSONSchemaType } from 'ajv';
+import betterAjvErrors from 'better-ajv-errors';
+import { ajv } from '../../ajv';
+import { Target } from '../types';
+
+export type AthenaTarget = {
+    type: 'athena';
+
+    database: string;
+    schema: string;
+    aws_access_key_id: string;
+    aws_secret_access_key: string;
+    s3_staging_dir: string;
+    region_name: string;
+    catalog: string;
+    work_group?: string;
+};
+
+export const athenaSchema: JSONSchemaType<AthenaTarget> = {
+    type: 'object',
+    properties: {
+        type: {
+            type: 'string',
+            enum: ['athena'],
+        },
+        schema: {
+            type: 'string',
+        },
+        s3_staging_dir: {
+            type: 'string',
+        },
+        region_name: {
+            type: 'string',
+        },
+        aws_access_key_id: {
+            type: 'string',
+        },
+        aws_secret_access_key: {
+            type: 'string',
+        },
+        database: {
+            type: 'string',
+        },
+        catalog: {
+            type: 'string',
+        },
+        work_group: {
+            type: 'string',
+            nullable: true,
+        },
+    },
+    required: ['type', 'schema', 's3_staging_dir', 'region_name'],
```

Comment:
I took the required parameters from [dbt-athena adapter docs](https://github.com/dbt-athena/dbt-athena?tab=readme-ov-file#configuring-your-profile). I wanted to keep consistent config with the adapter. But happy to change that, it can be derived from work group afterall.

Reply:
You are right, I checked the adatper implementation and `is_work_group_output_location_enforced` is used only for the data location, not for the staging result. As it is now for consistency we should do as you suggested, but this behvior can be changed in the future. In case of enforced workgroup configuration the staging location it's not necessary. I can't resolve comments, but this comment it's resolved on my side.